src/components/common/Content/MainPageContent/MainPageContent.tsx

```diff
@@ -51,45 +46,57 @@ export const AppContent = () => {
 
 
         const renderContent = () => {
-            if (index === 0 || index === 1) {
+            if (data.sliderId === 0 || data.sliderId === 1) {
                 return (
-                    <SliderWithDescription sliderId={data.sliderId} label={data.label} key={`${key}-description`}/>
+                    <SliderWithDescription
+                        sliderId={data.sliderId}
+                        label={data.label}
+                        type={data.type}
+                        key={`${key}-description`}
+                    />
                 );
             }
-            if (index === 2 || index === 3) {
+            if (data.sliderId === 2 || data.sliderId === 3) {
                 return (
                     <div
-                        className="content-container__black-background-content start-column-top-flex"
+                        className="main-page-content-container__black-background-content start-column-top-flex"
                         key={key}
                     >
                         <SliderBlackBackground
                             sliderId={data.sliderId}
                             label={data.label}
+                            type={data.type}
                             key={`${key}-black-background`}
                         />
                         {index === 3 && <ProvidersBlackBackgroundGrid/>}
                     </div>
                 );
             }
-            if (index === 4 || index === 5) {
+            if (data.sliderId === 4 || data.sliderId === 5) {
                 return (
-                    <SliderGrid sliderId={data.sliderId} label={data.label} index={index} key={`${key}-description`}/>
+                    <SliderGrid
+                        sliderId={data.sliderId}
+                        label={data.label}
+                        index={index}
+                        type={data.type}
+                        key={`${key}-description`}
+                    />
```

Comment:
You replaced index hardcode to ids hardcode, but this doesn't make difference. Better declare type of item as TS enum